src/Tabs/TabsWrapper.php

```diff
@@ -132,9 +131,32 @@ function twSetup(tabsid) {
   twObject[tabsid].counter = 100;
   // Close icon: removing the tab on click
   nav.on("click", "span.icon-close", function() {
-    var panelId = $(this).parent().attr("href").substring(1);
-    top.restoreSession();
-    twCloseTab(tabsid, panelId);
+    const self = $(this);
+    const closeTab = function() {
+        const panelId = self.parent().attr("href").substring(1);
+        top.restoreSession();
+        twCloseTab(tabsid, panelId);
+    }
+
+    const closeSoap = function() {
+        top.isSoapEdit = false;
+        closeTab();
+    }
+
+    if (self[0].id === 'SOAP' && top.isSoapEdit === true) {
+        dlgopen('', '', 450, 125, '', '<div class="text-danger">Warning</div>', {
```

Comment:
translate (note this is different than in above smarty context):
```
dlgopen('', '', 450, 125, '', '<div class="text-danger"><?php echo xla("Warning"); ?></div>', {
```

Reply:
Hi @bradymiller 

Here, The title will disappear, I guessing in here it using `<<<EOD` and `EOD`

![image](https://user-images.githubusercontent.com/22230889/78653978-7812b000-78f6-11ea-9352-63111e257dfb.png)

Reply:
man, EOD...
your gonna have to look up how to call php functions in EOD (there may be examples of this in OpenEMR codebase, but not sure)

Reply:
Sounds like a huge project 🤣, I will try to fix it.
Thank you.